drivers/sensor/tmag5273/Kconfig

```diff
@@ -0,0 +1,364 @@
+# TMAG5273 Low-Power Linear 3D Hall-Effect Sensor configuration options
+
+# Copyright (c) 2022 deveritec GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig TMAG5273
+	bool "TMAG5273 3D Hall-Effect Sensor"
+	depends on DT_HAS_TI_TMAG5273_ENABLED
+	select I2C
+	help
+		Enable driver for TMAG5273 I2C-based 3d Hall-Effect sensor.
+
+if TMAG5273
+
+comment "measurement"
+
+config TMAG5273_MAGNETIC_CH_X_EN
+	bool "Measure x-axis"
+	default y
+	select TMAG5273_MAGNETIC_EN
+
+config TMAG5273_MAGNETIC_CH_Y_EN
+	bool "Measure y-axis"
+	default y
+	select TMAG5273_MAGNETIC_EN
+
+config TMAG5273_MAGNETIC_CH_Z_EN
+	bool "Measure z-axis"
+	default y
+	select TMAG5273_MAGNETIC_EN
+
+config TMAG5273_MAGNETIC_EN
+	bool
```

Comment:
Shouldn't this have a prompt as well?

Reply:
The idea here is to define a config-variable which at the same time
- can be used to check for the measurement range in the Kconfig (see `TMAG5273_MEAS_RANGE_XYZ_CHOICE`), and
- remove some maybe unneeded code during compile time.